.travis.yml

```diff
@@ -4,3 +4,4 @@ after_success:
 node_js:
   - '6'
   - '7'
```

Comment:
Let's remove v7? Since it's not an LTS release.

Reply:
That seems reasonable; done.